examples/softirq-latency-net-rx.bpf.c

```diff
@@ -4,14 +4,21 @@
 
 // Loosely based on https://github.com/xdp-project/xdp-project/blob/master/areas/latency/softirq_net_latency.bt
 
-// 20 buckets for latency, max range is 0.5s .. 1.0s
-#define MAX_LATENCY_SLOT 21
+// 30 buckets for latency, max range is 0.5s .. 1.0s
+#define MAX_LATENCY_SLOT 31
 
 #define check_net_rx(vec_nr)                                                                                           \
     if (vec_nr != NET_RX_SOFTIRQ) {                                                                                    \
         return 0;                                                                                                      \
     }
 
+/* This provide easy way to disable measuring 'runtime'.
+ * This avoids hooking 'softirq_exit' as it can be expensive and for NET_RX
+ * this isn't the right hook as runtime is affected by NAPI packet bulking.
+ */
+#define CONFIG_MEASURE_RUNTIME	1
+//#undef CONFIG_MEASURE_RUNTIME
```

Comment:
Let's drop this line. I think one can simply comment the line above to achieve the same outcome.